packages/nest/src/interfaces/notifire-options.interface.ts

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface NotifireOptions {
+  //
+  // This interface describes the options you want to pass to
+  // NotifireModule.
+  //
+  providers: Array<any>;
```

Comment:
Can we import the IProvider and ITemplate interface from @notifire/core here ?

Reply:
Yes, we can.